assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -38,6 +38,10 @@ function useUploadVideoFrame({ updateMediaElement }) {
   }));
   const setProperties = useCallback(
     (id, properties) => {
+      // properties is a function in this case, which will carry the poster
+      // value in the return object.
```

Comment:
I'm not sure what this is saying. `properties` is a function whose return value is the video poster image?

Reply:
I am trying to describe this logic: 
```
({ resource }) => ({
        resource: {
          ...resource,
          posterId,
          poster,
          ...newSize,
        },
      })
```
as this is what properties really is😅. But I agree to remove these 2 lines of comments.